index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite App</title>
```

Comment:
Should this looks a bit more like the [old `public/index html` file](https://github.com/OpenSourceRaidGuild/website/blob/970b60a45aed7ea52e4883d9890fc3181bd2ed58/public/index.html)?

Reply:
ahahaha Oops

Reply:
Fixed Title "OSRG"